v2/cli/init_command.go

```diff
@@ -62,6 +62,13 @@ func initialize(c *cli.Context) error {
 		return err
 	}
 
+	if fields.GitCloneURL != "" {
+		err = core.ValidateGitCloneURL(fields.GitCloneURL)
+		if err != nil {
+			return err
+		}
```

Comment:
```
if fields.GitCloneURL != "" {
	if err = core.ValidateGitCloneURL(fields.GitCloneURL); err != nil {
		return err
	}
}
```

Reply:
Combine assignments and error-checking conditionals whenever you can. To be clear, the first version wasn't bad, but the revision I am suggesting is very clear and concise that we're handling errors related to a particular function call.